### DISCUSSION TITLE
Uninstall Homebrew from wrong directory

### DISCUSSION THREAD

**Discussion by almccombs** [Homebrew/discussions#3625] [Everyday usage] [answered]
I installed Homebrew in the wrong directory (using git clone) and would like to remove it for a clean install.  The official script only works if it's in the standard /opt/homebrew directory.  How do I uninstall?

**Comment by carlocab** [comment, marked as answer]
Just delete the directory your clone is in.

**Reply by almccombs** [reply]
@carlocab Thanks.

**Reply by jktam** [reply]
For a clean uninstallation do I need to remove an entry from $PATH or anything else?
